### PR TITLE
refactor: Do not track statik.go in git

### PR DESCRIPTION
**Pull request by jsuchome**
The file gets generated on every compilation, there's no need to
store it in git repository.

Fixes https://github.com/fuseml/fuseml/issues/35

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,22 +37,22 @@ compress:
...
+test-acceptance-traefik: embed_files
 	@./scripts/test-acceptance.sh -- -serve=deployment
 
-test-acceptance-knative:
+test-acceptance-knative: embed_files
 	@./scripts/test-acceptance.sh -- -serve=knative
 
-test-acceptance-kfserving:
+test-acceptance-kfserving: embed_files
 	@./scripts/test-acceptance.sh -- -serve=kfserving
```
Comment (flaviodsr) [inline Makefile:50]: Do we need to call embed files here? the test calls `build`, which already includes embed_files.

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jsuchome) [reply Makefile:50]: Where they call `build`?

Comment (flaviodsr) [reply Makefile:50]: see: https://github.com/fuseml/fuseml/blob/main/acceptance/acceptance_suite_test.go#L211

Comment (jsuchome) [reply Makefile:50]: But that's probably too late because some dependency required statik earlier. Let me show you by removing it

Comment (jsuchome) [reply Makefile]: Yep. The file imports `helpers` which means also `helpers/embedded_files.go` which requires built `statik` content...
https://github.com/fuseml/fuseml/pull/34/checks?check_run_id=2138381748

Comment (flaviodsr) [reply Makefile:50]: ok, thanks for checking 😃

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**